packages/agoric-cli/test/agops-oracle-smoketest.sh

```diff
@@ -1,5 +1,8 @@
 #!/bin/sh
 
+# "agoric follow" steps need to be cancelled manually, so these steps should be
+# copy-pasted rather than run as a script for best results.
```

Comment:
I don't quite understand what the problem is. Would `exec agoric follow` do the trick ?

Reply:
I'm not seeing any difference in behavior adding `exec` in front. It basically just hangs when it gets here:
![image](https://user-images.githubusercontent.com/8848650/210894872-8752bb19-a50e-468c-a7e5-b1a006be3833.png)